cmd/new.go

```diff
@@ -350,10 +351,16 @@ func installDependencies() error {
 		return nil
 	}
 
+	// Buffer STDERR so we can output it if there is an error.
+	var errBuffer bytes.Buffer
+	c.Stderr = &errBuffer
+
 	// Run the command.
-	c.Stdout = os.Stdout
-	c.Stderr = os.Stderr
-	return c.Run()
+	err = c.Run()
+	if err != nil {
+		fmt.Fprint(os.Stderr, errBuffer.String())
```

Comment:
Do we need to capture both stdout and stderr into a single stream and return the whole thing in case of error?  From the examples you shared, it feels like `stderr` alone may not include sufficient context to understand the failure.

Reply:
`npm` it doesn't write anything to `stdout` when I buffer it (you can see this by running `npm install 2>&1 log.txt`).

For `pip` it only adds two extra lines:

```
Looking in indexes: https://pypi.org/simple, https://pypi.pulumi.com/simple
Collecting pulumi>=0.16.0 (from -r requirements.txt (line 1))
```

But I'll still capture both `stdout` and `stderr` into a single buffer and write it all to `stderr` upon failure.